modules/site-health/audit-enqueued-assets/load.php

```diff
@@ -0,0 +1,232 @@
+<?php
+/**
+ * Module Name: Audit Enqueued Assets
+ * Description: Adds a CSS and JS resource checker in Site Health checks.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Require helper functions.
+ */
+require_once __DIR__ . '/helper.php';
+
+/**
+ * Audit enqueued scripts in is_front_page(). Ignore /wp-includes scripts.
+ *
+ * It will save information in a transient for 12 hours.
+ *
+ * @since 1.0.0
+ */
+function perflab_aea_audit_enqueued_scripts() {
+	if ( ! is_admin() && is_front_page() && current_user_can( 'view_site_health_checks' ) && false === get_transient( 'aea_enqueued_front_page_scripts' ) ) {
+		global $wp_scripts;
+		$enqueued_scripts = array();
+
+		foreach ( $wp_scripts->queue as $handle ) {
+			$src = $wp_scripts->registered[ $handle ]->src;
+			if ( $src && ! strpos( $src, 'wp-includes' ) ) {
+				$enqueued_scripts[] = array(
+					'src'  => $src,
+					'size' => perflab_aea_get_resource_file_size( perflab_aea_get_path_from_resource_url( $src ) ),
+				);
+			}
+		}
+		$expiration = apply_filters( 'perflab_aea_enqueued_front_page_scripts_expiration_in_seconds', 12 * HOUR_IN_SECONDS );
```

Comment:
Following up on the other comment thread in https://github.com/WordPress/performance/pull/54#discussion_r776510603, since we don't have a use-case for it in mind, I suggest we remove it? We can always add it in later if something comes up where it's useful, but for maintenance it's better not to arbitrarily add filters for something just because we can.

If we think there is a use-case for it, I think at least we should not have multiple filters for this in the different functions, since they are _way_ too specific, which is also clear by looking at the very long filter name. Again, I'd prefer to remove it, but if you feel strongly that we should keep it, maybe let's instead introduce a single new function like `perflab_aea_get_transient_expiration()` which applies this filter in a consistent way throughout this module (e.g. the filter could have its name simplified to `perflab_aea_transient_expiration`). Then the code here could be simplified to
```suggestion
		$expiration = perflab_aea_get_transient_expiration();
```

Also, for any of these filters, if we keep them, they still need to receive a doc-block, which is currently missing.

Reply:
Ok, agree, let's remove it for simplicity since we don't have a real use case here, and we want to roll this out soon.